<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="TATRC-17323" ignored="0" name="Admin - Simulation - Node Connector - Add/Edit/Delete/ErrorCheck" tests="1" failures="1" timestamp="07 Nov 2019 13:24:43 GMT" time="48.571" errors="0">
  <testcase name="execute" time="47.683" classname="testCases.Admin.Simulation.NodeConnectorAddEditDelete">
    <failure type="java.lang.AssertionError" message="Errors - Node Connector Already Exists expected [Connector Type already exists.] but found []">
      <![CDATA[java.lang.AssertionError: Errors - Node Connector Already Exists expected [Connector Type already exists.] but found []
	at testCases.Admin.Simulation.NodeConnectorAddEditDelete.performErrorValidation(NodeConnectorAddEditDelete.java:261)
	at testCases.Admin.Simulation.NodeConnectorAddEditDelete.execute(NodeConnectorAddEditDelete.java:93)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- execute -->
</testsuite> <!-- Admin - Simulation - Node Connector - Add/Edit/Delete/ErrorCheck -->
